src/go/rpk/pkg/cli/cmd/topic/consume.go

```diff
@@ -499,17 +853,49 @@ A key length as four big endian bytes, and the key as hex:
 A little endian uint32 and a string unpacked from a value:
     -f '%v{unpack[is$]}'
 
-MISC
+OFFSETS
 
 The --offset flag allows for specifying where to begin consuming, and
-optionally, where to stop consuming:
-
-    start    consume from the beginning
-    end      consume from the end
-    +NNN     consume NNN after the start offset
-    -NNN     consume NNN before the end offset
-    N1-N2    consume from N1 to N2
-
-If consuming a range of offsets, rpk does not currently quit after it has
-consumed all partitions through the end range.
+optionally, where to stop consuming. The literal words "start" and "end"
+specify consuming from the start and the end.
+
+    start     consume from the beginning
+    end       consume from the end
+    :end      consume until the current end
+    +oo       consume oo after the current start offset
```

Comment:
Is `oo` a placeholder?

Reply:
Not quite a placeholder, I thought it might look better than NNN but I'm not entirely sold on either.

Reply:
What do you think? Resolving for now but I can change to whatever the best is (ddd? dd? I thought oo may be better for signifying "offset").